src/main/java/frc/robot/Constants.java

```diff
@@ -147,6 +147,19 @@ public static class LEDConstants {
   public static class AutoAlignConstants {
     public static final double REEF_OFFSET_RIGHT = Units.inchesToMeters(1);
     public static final double REEF_OFFSET_LEFT = Units.inchesToMeters(12);
+
+    // needs to be tuned
+    public static final double ROBOT_ROTATION_P = 5; // 11.507 from rotation sys-id @PF 1/13
+    public static final double ROBOT_ROTATION_I = 0;
+    public static final double ROBOT_ROTATION_D = 0; // 0.10877 from rotation sys-id @PF 1/13
+
+    public static final double ROBOT_POSITION_P = 20; // too high
```

Comment:
Change this back to the same as the PP one now that you've added the velocity setpoint.